packages/flutter/test/widgets/widget_inspector_test.dart

```diff
@@ -1545,18 +1546,48 @@ class TestWidgetInspectorService extends Object with WidgetInspectorService {
         matchesGoldenFile('inspector.clipRect_debugPaint.png'),
       );
 
+      final Element clipRect = find.byType(ClipRRect).evaluate().single;
```

Comment:
I noticed when I re-applied this that I was missing a test that actually tested end to end instead of just testing the
InspectorService.screenshot method.